Formula/infer.rb

```diff
@@ -164,7 +132,7 @@ class PassingTest {
       }
     EOS
 
-    shell_output("#{bin}/infer --fail-on-issue -P -- javac FailingTest.java", 2)
-    shell_output("#{bin}/infer --fail-on-issue -P -- javac PassingTest.java")
+    shell_output("#{bin}/infer --fail-on-issue -P -- #{java_home}/bin/javac FailingTest.java", 2)
+    shell_output("#{bin}/infer --fail-on-issue -P -- #{java_home}/bin/javac PassingTest.java")
```

Comment:
Do these produce any output we can check with `assert_equal` or `assert_match`? Same with the C tests above.

Reply:
So, I did this, but I don't know what the correct style is for Homebrew -- I just copied the whole output.

What's the "norm" for this kind of thing?